.gitignore

```diff
@@ -100,3 +100,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# uv
+uv.lock
```

Comment:
I think we should rather check in `uv.lock` (https://docs.astral.sh/uv/concepts/projects/layout/#the-lockfile)